app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx

```diff
@@ -0,0 +1,142 @@
+import LineLoader from "../../components/Main/LineLoader/LineLoader";
+import { useCustomPanelState } from "../../state/customPanel/CustomPanelStateContext";
+import { useQueryState } from "../../state/query/QueryStateContext";
+import "../CustomPanel/style.scss";
+import Alert from "../../components/Main/Alert/Alert";
+import classNames from "classnames";
+import useDeviceDetect from "../../hooks/useDeviceDetect";
+import { useRef } from "react";
+import CustomPanelTabs from "../CustomPanel/CustomPanelTabs";
+import { DisplayTypeSwitch } from "../CustomPanel/DisplayTypeSwitch";
+import QueryConfigurator from "../CustomPanel/QueryConfigurator/QueryConfigurator";
+import WarningLimitSeries from "../CustomPanel/WarningLimitSeries/WarningLimitSeries";
+import { useFetchExport } from "./hooks/useFetchExport";
+import { useSetQueryParams } from "../CustomPanel/hooks/useSetQueryParams";
+import { DisplayType } from "../../types";
+import Hyperlink from "../../components/Main/Hyperlink/Hyperlink";
+import { CloseIcon } from "../../components/Main/Icons";
+import Button from "../../components/Main/Button/Button";
+
+const RawQueryPage: FC = () => {
+  useSetQueryParams();
+  const { isMobile } = useDeviceDetect();
+
+  const { displayType } = useCustomPanelState();
+  const { query } = useQueryState();
+
+  const [hideQuery, setHideQuery] = useState<number[]>([]);
+  const [hideError, setHideError] = useState(!query[0]);
+  const [showAllSeries, setShowAllSeries] = useState(false);
+  const [showPageDescription, setShowPageDescription] = useState(true);
+
+  const {
+    data,
+    error,
+    isLoading,
+    warning,
+    queryErrors,
+    setQueryErrors,
+    abortFetch
+  } = useFetchExport({ hideQuery, showAllSeries });
+
+  const controlsRef = useRef<HTMLDivElement>(null);
+
+  const showError = !hideError && error;
+
+  const handleHideQuery = (queries: number[]) => {
+    setHideQuery(queries);
+  };
+
+  const handleRunQuery = () => {
+    setHideError(false);
+  };
+
+  const handleHidePageDescription = () => {
+    setShowPageDescription(false);
+  };
+
+  return (
+    <div
+      className={classNames({
+        "vm-custom-panel": true,
+        "vm-custom-panel_mobile": isMobile,
+      })}
+    >
+      <QueryConfigurator
+        label={"Time series selector"}
+        queryErrors={!hideError ? queryErrors : []}
+        setQueryErrors={setQueryErrors}
+        setHideError={setHideError}
+        stats={[]}
+        isLoading={isLoading}
+        onHideQuery={handleHideQuery}
+        onRunQuery={handleRunQuery}
+        abortFetch={abortFetch}
+        hideButtons={{ traceQuery: true, disableCache: true }}
+        includeFunctions={false}
+      />
+      {showPageDescription && (
+        <Alert variant="info">
+          <div className="vm-explore-metrics-header-description">
+            <p>
+              This page provides a dedicated view for querying and displaying raw data directly from VictoriaMetrics.
```

Comment:
```suggestion
              This page provides a dedicated view for querying and displaying raw samples from VictoriaMetrics.
```
Please link `raw samples` to https://docs.victoriametrics.com/keyconcepts/#raw-samples

Reply:
Please add `It expects only time series selector as a query argument.`
please wrap `time series selector` with https://docs.victoriametrics.com/keyconcepts/#filtering link.